src/openforms/submissions/urls.py

```diff
@@ -15,4 +15,9 @@
         SubmissionAttachmentDownloadView.as_view(),
         name="attachment-download",
     ),
+    path(
+        "<int:submission_id>/evaluated-logic-logs",
+        LogsEvaluatedLogic.as_view(),
+        name="logs-evaluated-logic",
+    )
```

Comment:
This url belongs in the `SubmissionAdmin.get_urls` instead of in the "public" url patterns.